tests/transients.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
@targetsm if you want the test to run every time the CI runs, it has to be called `*_test.py`. I'd recommend putting it in `tests/transformations/transient_reuse_test.py`